src/api/app/controllers/webui/user_controller.rb

```diff
@@ -225,24 +225,6 @@ def tokens
     render json: list_users(params[:q], true)
   end
 
-  def notifications
```

Comment:
Looks like you have to update the routes then: https://github.com/openSUSE/open-build-service/blob/master/src/api/config/routes.rb#L61

Reply:
This is in a `controller 'webui/feeds'` and this is the `controller 'webui/user'`

Reply:
ah, right